tests/quick_tests/run.cmake

```diff
@@ -21,12 +21,11 @@ SEPARATE_ARGUMENTS(ALL_TESTS)
 
 EXECUTE_PROCESS(COMMAND ${CMAKE_CTEST_COMMAND} --force-new-ctest-process --output-on-failure -O quicktests.log RESULT_VARIABLE res_var)
 
-if(NOT "${res_var}" STREQUAL "0")
-  MESSAGE( "
-
-*******************************     WARNING     *******************************
-
-Some of the tests failed!
+IF(NOT "${res_var}" STREQUAL "0")
+  MESSAGE("
+***************************************************************************
+**                 Error: Some of the quick tests failed.
```

Comment:
I originally included the two stars at the end but then removed them when I noticed that the other 'starry comments' (i.e., the output of `macro_deal_ii_invoke_autopilot.cmake` did not have them. Hence I just tried to make things consistent.

Nonetheless, starred boxes are nice, so if you don't mind I will fix all of them in a follow up PR.

I appreciate the eye for detail :)